spec/features/saml/ial1_sso_spec.rb

```diff
@@ -120,7 +120,8 @@
     it 'it immediately returns to the SP after signing in again' do
       click_continue
 
-      visit sign_out_url
+      # Don't want to 'remember device'
+      Capybara.reset_session!
```

Comment:
Looks like this pair of lines is repeated a handful of times throughout the specs, WDYT of a simple helper method like `sign_out_and_forget_device!`?

Reply:
I think we normally use this to simulate visiting from a different browser, so we could possibly go for something like "visit_idp_from_a_different_browser`

Reply:
I thought that was the `perform_in_browser(:one)` etc etc syntax?

Reply:
Hmm not a bad idea, but I think in terms of clarity, the comment suffices - and although the blurb is repeated throughout, it's really only one method being called in each spot. Concerning readability/simplicity, yes, the comment is repeated with every occurrence (for clarity), but I don't personally see a need for a new method, since it's only really doing one single thing. @jmhooper's call, maybe?

Reply:
Lol I responded before reloading this page to see the rest of this conversation...

Reply:
Actually, yeah I can see how this pattern is more idiomatic for our feature tests and that's worthwhile, so I'll go ahead and put it in a method that reads like the others. 😄